modules/preview-react/segmented-control/stories/SegmentedControl.stories.mdx

```diff
@@ -38,62 +37,86 @@ yarn add @workday/canvas-kit-preview-react
 
 ### Basic Example
 
-In a basic example of a `SegmentedControl` there are four icon buttons in group. Each button is
-paired with a tooltip to explain the button function in text. Default values of `SegmentedControl`
-are specified for size (as `medium`) and orientation (as `horizontal`). All buttons should have the
-same type and only one of them can be active at a time.
-
 `SegmentedControl` includes a container `SegmentedControl` component and the following
-subcomponents: `SegmentedControl.List` which contains a `SegmentedControl.Item`.
+subcomponents: `SegmentedControl.List` and `SegmentedControl.Item`.
+
+The example below contains a `SegmentedControl` with four icon-only buttons. Each button is rendered
+using a `SegmentedControl.Item` and is paired with a tooltip describing the button's function. Only
+one button can be active at a time.
```

Comment:
@RayRedGoose Can you update the Basic example to show how a user would extract the currently selected value from the Segmented Control? The example has four buttons for Table, List, Detail, and Diagram -- how do I (as a developer) get the currently selected value? We can just print it out in a paragraph below the example like we do for [Menu](https://5d854c26ba934e0020f5e98a-khqkejkfmy.chromatic.com/?path=/docs/components-popups-menu-react--basic).

Reply:
Good point! Added.